platform-marvel/README.md

```diff
@@ -15,6 +15,8 @@ A Plataforma Marvel é um projeto inovador que oferece aos usuários uma experi
 - **Exploração de HQs:** Navegue por uma vasta coleção de histórias em quadrinhos da Marvel, com resumos, ilustrações e detalhes sobre autores e artistas.
 
 - **Sistema de Login:** Os usuários podem criar uma conta e fazer login para acessar recursos exclusivos, como favoritos e recomendações personalizadas.
+  
+- - **Feedback  com nota:** Os usuários podem dar uma reação que será analizada por uma NLP de analise de sentimentos com a lib TextBlob do python.
 
 ## Como Executar
```

Comment:
Exemplo de como usar a lib TextBlob + googletrans, para analises de sentimentos:

# comandos = pip install textblob googletrans==4.0.0-rc1
# comandos = python -m textblob.download_corpora


from textblob import TextBlob 
from googletrans import Translator



"""
resultados(+) => Respostas Positivas 👍
resultados(-) => Respostas Negativas 👎

"""

# EXEMPLO-1
frase1 = 'Python is great for machine learning'
frase2 = 'Python is Horrible for machine learning'
tb_1 = TextBlob(frase1)
tb_2 = TextBlob(frase2)

print(tb_1.sentiment.polarity)
print(tb_2.sentiment.polarity)

# EXEMPLO-2
# Preenchimento do texto a ser analisado (Em português)
text = 'Gregório é um programador   incrivel!'

# Criando objeto TextBlob para tratar o texto em português e traduzir para inglês (idioma suportado pelas libs)
translator = Translator()
translated = translator.translate(text, src='pt', dest='en')
text_translated = translated.text

# Fazendo análise de polaridade usando a lib TextBlob
blob = TextBlob(text_translated)
p = blob.sentiment.polarity

if p < 0:
    result = 'Negativo'
elif p > 0:
    result = 'Positivo'
elif p == 0:
    result = 'Neutro'
else:
    result = 'Indefinido'

print(f'Resultado: Sentimento {result}')
print(f'Polaridade: {p}')


`saída do código:`
0.8
-1.0
Resultado: Sentimento Positivo
Polaridade: 1.0